05_stringer/mohitnag/main_test.go

```diff
@@ -0,0 +1,38 @@
+package main
+
+import (
+	"bytes"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestIpaddr(t *testing.T) {
+	testData := []struct {
+		Scenario string
+		input    IPAddr
+		expected string
+	}{
+		{"Scenario One", [4]byte{127, 0, 0, 1}, "127.0.0.1"},
```

Comment:
I guess this comes down to personal preference and how you want your tests to output when running / failing, but I think it is nicer to give a descriptive name rather than "Scenario x".

Something more like "localhost", "empty", "Two Values" etc to give it a little bit more when outputting a failure. Can you please update this?

Reply:
Done